tests/test_new.diff

```diff
@@ -112,4 +112,67 @@ index 0000000..7cccc5b
 +++ b/src/tests/example_file.sql
 @@ -0,0 +1,2 @@
 +-- TODO Select all:
-+SELECT * FROM Products;
\ No newline at end of file
++SELECT * FROM Products;
+diff --git a/tests/example_file.tex b/src/tests/example_file.tex
+new file mode 100644
+index 0000000..7cccc5b
+--- /dev/null
++++ b/src/tests/example_file.tex
+@@ -0,0 +1,2 @@
++% TODO Add in preamble details
++\begin{document}
++  \begin{comment}
++     TODO This document needs content
++     label: urgent
++  \end{comment}
++\end{document}
+diff --git a/tests/example_file.jl b/src/tests/example_file.jl
+new file mode 100644
+index 0000000..7cccc5b
+--- /dev/null
++++ b/src/tests/example_file.jl
```

Comment:
@alstr FYI this is the diff for the julia failing test (the add/remove diffs are the same actual julia content). 

Not quite sure why this would be parsed into 3 todos.

Reply:
I think it's because the line comment marker `#` appears within the block comment marker `#=`/`=#`. I haven't tested this but I've opened issue #102. Probably need to only match a marker if it is an exact match.